### PR TITLE
Add cleanup for PolkadotJS and network timeout

### PR DESCRIPTION
**Pull request by crystalin**
Leaving APIs and Network timeout timer on prevents NodeJS to exit properly.
This PR adds cleanup for those

### CODE REVIEW

File: src/network.ts [hunk 1]
```diff
@@ -140,6 +140,12 @@ export class Network {
   }
 
   async stop() {
+    // Cleanup all api instances
+    for (const para of Object.values(this.paras))
```
Comment (pepoviola) [inline src/network.ts:144]:
```suggestion
    for (const node of Object.values(this.nodesByName))
      node.apiInstance?.disconnect();
```

**Comment by crystalin** [comment]
@pepoviola it should be good now

File: src/network.ts [hunk 2]
```diff
@@ -140,6 +140,11 @@ export class Network {
   }
 
   async stop() {
+    // Cleanup all api instances
+    for (const node of Object.values(this.nodesByName))
+      node.apiInstance?.disconnect();
+    for (const relay of this.relay) 
+      relay.apiInstance?.disconnect()
```
Comment (pepoviola) [inline src/network.ts:147]:
```suggestion
```

`nodesByName` contains all nodes 👍

**Comment by crystalin** [comment]
Thanks I didn't know

**Review by pepoviola — APPROVED** [review]

**Comment by crystalin** [comment]
FYI: With this PR (and the previous ones that got merged) we are able to test in our CI with Zombienet

**Comment by pepoviola** [comment]
Thanks @crystalin!!

**Pull request merged**